fe/fe-core/src/main/java/org/apache/doris/nereids/memo/Group.java

```diff
@@ -257,6 +257,65 @@ public int removeParentExpression(GroupExpression parent) {
         return parentExpressions.size();
     }
 
+    /**
```

Comment:
why move the position of these functions?

Reply:
put correlative function together for easy reading